server/src/main/java/org/opensearch/index/query/NestedQueryBuilder.java

```diff
@@ -130,6 +130,13 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeBoolean(ignoreUnmapped);
     }
 
+    /**
```

Comment:
Thanks @psmagin for the change. 
Could you add a simple unit test and we should be good to go.

Reply:
@saratvemulapalli, added